website/static/js/registrationEditorExtensions.js

```diff
@@ -177,7 +177,16 @@ var Uploader = function(question) {
         question.value(question.formattedFileList());
     });
 
+    self.UPLOAD_LANGUAGE = 'You may attach up to 5 files to this question. You may attach files that you already have' +
+        'in this OSF project, or upload a new file from your computer. Uploaded files will automatically be added to this project' +
+        'so that they can be registered.';
```

Comment:
There are some missing spaces between each of these string concatenations:
![screen shot 2016-04-27 at 12 27 16 pm](https://cloud.githubusercontent.com/assets/7131985/14859978/937f6d6c-0c74-11e6-8c1b-533e0a148c85.png)